tmail-backend/webadmin/webadmin-oidc-backchannel/src/test/java/com/linagora/tmail/webadmin/OidcBackchannelLogoutRoutesTest.java

```diff
@@ -163,7 +162,7 @@ void postShouldReturn200WhenSidIsEmpty() {
         .when()
             .post()
         .then()
-            .statusCode(200);
+            .statusCode(400);
```

Comment:
According to the CI, there is an issue here: https://james-jenkins.lin-saas.com/job/Tmail%20build/job/PR-1969/1/testReport/junit/com.linagora.tmail.webadmin/OidcBackchannelLogoutRoutesTest/postShouldReturn400WhenSidIsEmpty/